services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -357,6 +354,13 @@ class SchedulerData(CommonServiceDetails, DynamicSidecarServiceLabels):
     proxy_service_name: Optional[str] = Field(
         None, description="service name given to the proxy"
     )
+    dynamic_sidecar_node_id: Optional[str] = Field(
```

Comment:
so if you have that value in SchedulerData, why is it in separate variables in other functions?
Just do a Ctrl+Shift+F of ```dynamic_sidecar_node_id```
Also, why repeat dynamic_sidecar? Basically I would suggest something like ```docker_node_id``` or ```host_id```

Reply:
Let me refactor it properly, since I've just introduced this change. Also will shorten the name